xarray/tests/test_groupby.py

```diff
@@ -2289,3 +2296,20 @@ def test_resample_cumsum(method: str, expected_array: list[float]) -> None:
     actual = getattr(ds.foo.resample(time="3M"), method)(dim="time")
     expected.coords["time"] = ds.time
     assert_identical(expected.drop_vars(["time"]).foo, actual)
+
+
+def test_groupby_binary_op_regression():
```

Comment:
```suggestion
def test_groupby_binary_op_regression() -> No e:
```

Annotate new tests plz :)